build.sbt

```diff
@@ -5,14 +5,16 @@ version := "1.0"
 
 lazy val root = (project in file(".")).enablePlugins(PlayJava)
 
-scalaVersion := "2.13.8"
+scalaVersion := "2.13.10"
 
 libraryDependencies ++= Seq(
   guice,
-  "dev.morphia.morphia" % "morphia-core" % "2.2.7",
-  "com.github.victools" % "jsonschema-generator" % "4.16.0",
-  "com.github.victools" % "jsonschema-module-jackson" % "4.16.0",
-  "com.github.victools" % "jsonschema-module-javax-validation" % "4.16.0"
+  "org.slf4j" % "slf4j-api" % "2.0.6",
+  "ch.qos.logback" % "logback-classic" % "1.4.5",
```

Comment:
slf4j and logback were not present before, why are they here now?

Reply:
responded in https://github.com/sircoderin/terra-management/pull/86